### DISCUSSION TITLE
Blocky port 53

### DISCUSSION THREAD

**Discussion by SimonMcN** [tteck/Proxmox#639] [Help Needed] [answered]
I may be delusional but does this work please ?
I have found that port 53 is blocked by systemd-resolved I have to do

systemctl stop systemd-resolved
systemctl disable systemd-resolved.service

before it will be able to accept queries on port 53 ?

**Comment by SimonMcN** [comment]
In my opinion this is an issue rather than a discussion.
The container was built from the script and it didn't work until I disabled the service ?

**Comment by SimonMcN** [comment]
There was already a discussion ongoing about blocky that I commented on with this btw.

**Comment by tteck** [comment, marked as answer]
As stated...
🚨 For Advanced Users.

**Comment by SimonMcN** [comment]
I appreciate that but it's in main and it doesn't work without more than just modifying the config file and there is no documentation stating the need to disable local dns name resolution.  If you don't believe it's an issue then go ahead and close it, but in 25 years in IT I don't know anyone who wouldn't say it's an issue ;-).

Thank you for your efforts though.  I guess as long as it's working for me it's fine ;-).

**Reply by tteck** [reply]
So, you're requesting that `systemctl stop systemd-resolved && systemctl disable systemd-resolved.service` be added to the script?

**Reply by SimonMcN** [reply]
That was my fix but I'm a new and don't know if that is the correct fix.  I would suspect that not installing systemd resolved would be better but I genuinely can't see how/if it got in to the build.

My request would be that you run the command, see if you have the same issue and recommend a solution.  It is possible it's just me, but I don't know how that would be.

Thanks for taking the time to reply.

**Reply by tteck** [reply]
Change made

**Reply by SimonMcN** [reply]
Thank you :-)
